graphics/blender/Portfile

```diff
@@ -8,7 +8,17 @@ version             2.83.0
 revision            0
 categories          graphics multimedia
 platforms           darwin
-license             {GPL-2+ OpenSSLException}
+# Note: Blender itself is licensed under GPL-2+ only. The other licenses
+# have been added because MacPorts' automatic license checking system
+# will flag Blender as being not distributable, because somewhere in
+# Blender's dependency tree, there is a package that does depend on
+# OpenSSL or graphviz. However, the MacPorts build of Blender specified
+# in this portfile is in no way dependent on OpenSSL or graphviz. This
+# mismatch, unfortunately, is unavoidable and outside of Blender's
+# control when using shared libraries. If Blender is built using all
+# internally-compiled dependencies, such licensing conflicts would not
+# be happening.
+license             {GPL-2+ OpenSSLException} EPL-1
```

Comment:
Is it possible to instead use `license_noconflict` here?

Reply:
I have no idea. I didn't know such a thing existed. Let me look into it, and report back.

Reply:
Yes, it looks like using `license_noconflict` is the right tool for the job. Thanks for pointing this out, like I said I didn't know such a thing existed. I'll modify my commit to start using it.

Reply:
Ok, I changed my commit to use `license_noconflict`. Hopefully I've added everything everything needed to resolve the licensing issues. If not, I might have to submit addition PRs.

Reply:
make sure there actually is an openssl exception in blender's tree, otherwise this is problematic and will have to be removed, and any macports-supplied files will have to be removed.

I didn't fully realize a license change was included in that minor version bump I committed for you a few days ago.

It's frustrating, but people are not kidding about these licenses, and people and groups have been sued for millions and millions of dollars for things like this.

Reply:
I actually removed the openssl exception entirely. As Christopher suggested above, it turns out that using `license_noconflict` is the more appropriate path to take. I even modeled my `license_noconflict` lines after what is already existing in another portfile: octave's.